README.md

```diff
@@ -1 +1,34 @@
-# hello_microverse-project
\ No newline at end of file
+<div align="center">
+
+  <h3><b>Hello Microverse</b></h3>
+
+</div>
+
+> **Hello Microverse** is a project meant to practice linters on html and css, GitHub_flow, pull_request and more in order to use the learned in all Microverse's Projects.
+
+---
+
+## 🛠 Built With <a name="built-with"></a>
+
+- HTML
+- CSS
+
+---
+
+## 🚀 Live Demo <a name="live-demo"></a>
```

Comment:
- [ ] Kindly remove the live demo section from your README since your project is yet/not deployed. you will add this section when you have a deployed link

Reply:
Hi @tufoinnkuo10 thank you!, so I should only remove the :

- [Live Demo Link](https://)

and leave the required header:

## 🚀 Live Demo <a name="live-demo"></a>

Reply:
@ITurres the whole section can be removed, including the live demo header